### PR TITLE
Add state to disk module

### PR DESCRIPTION
**Pull request by xenrox**
I prefer to hide my disk information until only a certain percentage is left. This would implement that functionality in a way like the CPU/memory modules use `state`.

### CODE REVIEW

File: src/modules/disk.cpp [hunk 1]
```diff
@@ -73,6 +73,7 @@ auto waybar::modules::Disk::update() -> void {
       ));
   }
   event_box_.show();
+  getState((stats.f_blocks - stats.f_bavail) * 100 / stats.f_blocks);
```
Comment (Alexays) [inline src/modules/disk.cpp:76]: Could save the operation into a variable in order to avoid computing it twice 😄

**Review by Alexays — CHANGES_REQUESTED** [review]

File: src/modules/disk.cpp (hunk 1 above)
Comment (xenrox) [reply src/modules/disk.cpp]: Done 😃

**Review by Alexays — APPROVED** [review]

**Comment by Alexays** [comment]
Thanks!
Can you update the GitHub wiki? ❤️

**Comment by xenrox** [comment]
> Can you update the GitHub wiki? ❤️

Done as well. Thanks for your work!

**Pull request merged**